src/app/nft-showcase/nft-showcase.component.ts

```diff
@@ -32,8 +35,11 @@ export class NftShowcaseComponent implements OnInit {
     this.globalVars = _globalVars;
   }
 
-  ngOnInit(): void {
-    this.loading = true;
+  showcaseComingSoon() {
+    return !this.loading && (!this.nftCollections || !has(this.nftCollections, length));
+  }
+
```

Comment:
we should be able to kill this function and just use the constants to define the conditional